qlib/DebugCmdLine.qm

```diff
@@ -255,13 +255,89 @@ public namespace DebugCmdLine {
                         },
                     ),
 
+                    'cmd': (
+                        'desc': 'send a command to the server and show the output',
+                        'action': sub(DebugCommandLine dcl, softlist path, softlist args) {
+                            #printf("path: %y args: %y\r\n", path, args);
+                            hash data;
+                            try {
+                                foreach string str in (args) {
+                                    any v = parse_to_qore_value(str);
+                                    if (v.typeCode() == NT_STRING && !data) {
+                                        data.cmd = v;
+                                        continue;
+                                    }
+                                    if (v.typeCode() != NT_HASH)
+                                        throw "CMD-ARG-ERROR", sprintf("arg must be a hash; got type %y instead", v.type());
+                                    data += v;
+                                }
+                                if (!data)
+                                    stdout.printf("syntax: cmd <hash>\r\n");
+                                else {
+                                    dcl.doCommand(data);
+                                }
+                            }
+                            catch (hash<ExceptionInfo> ex) {
+                                stdout.printf("ERROR: cmd cannot be parsed: %s: %s\r\n", ex.err, ex.desc);
+                            }
+                        },
+                    ),
+                    'frame': (
+                        'enum': list sub(DebugCommandLine dcl, softlist path, softlist args) {
```

Comment:
set frame is via new level '*',  (as pgm/set/*).  This way does not work as enum is for '*' only. Also '*': 'action' should implement set-frame-functionality